nixodmlconverter/convert.py

```diff
@@ -32,6 +32,10 @@
     -o              Specify name of output file. Only if one file is presented.
     -t              Convert only files of this type/with this extension.
                     Only, if a directory is presented.
+    -m              Conversion mode:
+                        - append (only for conversion to nix, standard setting)
+                        - overwrite (automatically for nix to odML conversion)
+                        - overwrite metadata (only change nix file's odML metadata)
```

Comment:
Since the conversion mode is now used in a command line context as well, the whitespace in `overwrite metadata` should be removed or replaced.